include/net/net_pkt.h

```diff
@@ -2086,8 +2086,8 @@ struct net_pkt *net_pkt_rx_alloc_with_buffer_debug(struct net_if *iface,
 /**
  * @brief Allocate an initialized net_pkt
  *
- * Note: for the time being, 2 pools are used. One for TX and one for RX.
- *       This allocator has to be used for TX.
+ * @details for the time being, 2 pools are used. One for TX and one for RX.
```

Comment:
As I mentioned before, I consider separate RX vs TX pools a great feature. If there's a desire to allow working with one common pool, that's fine, but separate RX vs TX pool config should be also kept.